### DISCUSSION TITLE
Can i use it with Android Jetpack Compose, not Kotlin Multiple Platform

### DISCUSSION THREAD

**Discussion by du-nt** [soil-kt/soil#124] [General]
**Please consider asking your question on [Github discussions](https://github.com/soil-kt/soil/discussions). The github issue tracker is for bugs and feature requests.**

**Comment by ogaclejapan** [comment]
Hi, @du-nt 

Is anything not working for you? 😄
You can use it with Android Jetpack Compose too by defining the library the same way as for KMP 👌  

Check out the guide here: https://docs.soil-kt.com/guide/getting-started.html
